File: Objects/Maintenance.java
package Objects;

public class Maintenance {
    private static int n = 0;
    private final int ID;
    private final String employee;
    private final String animalID;
    private final String process;
    private final String date;
    private final String comment;

    public Maintenance(String employee, String animalID, String process, String date, String comment) {
        this.employee = employee;
        this.animalID = animalID;
        this.process = process;
        this.date = date;
        this.comment = comment;
        n++;
        ID = n;
    }

    public String getDetails(){
        if(comment.equalsIgnoreCase("none")){
            return String.format("%nMaintenance number: #%d %nEmployee in charge: %s, date: %s, animal's ID: %s%nProcess: %s",ID,employee,date,animalID,process);
        }else{
            return String.format("%nMaintenance number: #%d %nEmployee in charge: %s, date: %s, animal's ID: %s%nProcess: %s%nObservations: %s" ,ID,employee,date,animalID,process,comment);
        }
    }

}

File: Modules/Objects/Maintenance.py
class Maintenance:
    n = 0
    __id = 0
    __employee = " "
    __animal_id = " "
    __process = " " 
    __date = " "
    __comment = " "

    def __init__(self, employee, animal_id, process, date, comment):
        self.__employee = employee
        self.__animal_id = animal_id
        self.__process = process
        self.__date = date
        self.__comment = comment
        n = n + 1
        self.__id = n

    def __get_details(self):
        return"\n Maintenance number: %d \nEmployee in charge: %s \nDate: %s \nAnimal's ID: %s \n Process: %s \nObservations: %s \n", self.__id, self.__employee, self.__date, self.__animal_id, self.__process, self.__comment 
    